doc/api/process.md

```diff
@@ -1642,6 +1642,40 @@ This function is only available on POSIX platforms (i.e. not Windows or
 Android).
 This feature is not available in [`Worker`][] threads.
 
+## process.nice(inc)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `inc` {integer} The new nice value for the process.
```

Comment:
We should include return value information here since this function *always* returns a value.

Reply:
Check. a099642629b36e908c5114227c4fd8e9c18cc12f